tests/unit/array/storage/elastic/test_add.py

```diff
@@ -0,0 +1,33 @@
+from docarray import Document
+from docarray.array.storage.base.helper import Offset2ID
+from docarray.array.storage.elastic.seqlike import SequenceLikeMixin
+
+
+class MockESMixin(SequenceLikeMixin):
+    def __delitem__(self):
+        pass
+
+    def __getitem__(self):
+        pass
+
+    def __setitem__(self):
+        pass
+
+
+def test_add_ids_that_already_exist():
+    es_seq_mixin = MockESMixin()
```

Comment:
I think a best solution is to come with `monkeypatch` here and path the `ESBackend` Mixin or functions and wori wht the `da.add()` interface so that is clear what is being tested and what not. Here we are doing a lot of private magic to make the test work, this should be done inside the Mocking class itself

Reply:
yep. instead I will change this using integration test, so that the main issue shows directly